### PR TITLE
greenhouse-ccloud: update pluginpresets

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml [hunk 1]
```diff
@@ -52,19 +72,32 @@ spec:
...
     overrides:
     - name: global.commonLabels
       value:
-        service: kvm
+        service: observability
     - name: kubeMonitoring.prometheus.prometheusSpec.externalLabels
       value:
         cluster: '{{ "{{ .Values.global.greenhouse.clusterName }}" }}'
-        cluster_type: compute
+        cluster_type: observability
```
Comment (viennaa) [inline system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml:84]: Why was this on `kvm` and `compute`?

Comment (richardtief) [reply system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml:84]: It's still there. Line 59 and 63. Git is doing some strange diff here.

Comment (viennaa) [reply system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml:84]: Ah my bad 👍

**Review by viennaa — COMMENTED** [review]
Bump the chart pls :)

**Review by viennaa — APPROVED** [review]
Thanks.

**Pull request merged**